mrf_apps/mrf_read.py

```diff
@@ -107,7 +107,7 @@
 
     if len(rasterElem.getElementsByTagName('Size')):
         sizeElem  = rasterElem.getElementsByTagName('Size')[0]
-        sizeAttrs = dict(sizeElem.attributes.items())
+        sizeAttrs = dict(list(sizeElem.attributes.items()))
```

Comment:
this seems to say:
sizeAttrs = sizeElem.attributes
Why take a dictionary, convert it to list and then back to a dictionary?  A deepcopy doesn't seem to be needed.

Reply:
I ran 2to3 on the script and this is something it auto corrected.  Should I put it back to original or leave it like this?

Reply:
I think the original was fine, but test.
Don't know why 2to3 insists on converting things to list, looks like "better safe than sorry" principle.

Reply:
I reverted it back to original